stacks/java-springboot/devfile.yaml

```diff
@@ -17,7 +17,8 @@ starterProjects:
 components:
   - name: tools
     container:
-      image: quay.io/eclipse/che-java11-maven:next
+      image: registry.access.redhat.com/ubi8/openjdk-11:latest
+      command: ["tail", "-f", "/dev/null"]
```

Comment:
Can we use UDI instead for the sake of unification https://github.com/devfile/developer-images ?